scripts/benchmark_all.sh

```diff
@@ -12,7 +12,17 @@ then
 fi
 
 benchmarks='hyriseBenchmarkTPCH hyriseBenchmarkTPCDS hyriseBenchmarkTPCC hyriseBenchmarkJoinOrder'
-num_mt_clients=50
+warmup_seconds=2
+runs=100
+
+# Setting the number of clients used for the multi-threaded scenario to the machine's core count.
+# This only works for macOS and Linux.
+output="$(uname -s)"
+case "${output}" in
+    Linux*)     num_mt_clients="$(lscpu -p | egrep -v '^#' | grep '^[0-9]*,[0-9]*,0,0' | wc -l)";;
```

Comment:
This returns `56` on Nemea, no matter if the shell is bound to a single numa node or not.

Reply:
Yes, that's what we wanted to achieve in the other project. But now that you bring it up, I am not even sure what makes most sense here.
For us, the purpose was to not use more clients/cores than on there are on a single machine. That makes sense since we still neglect NUMA. But I am not sure we should use it here. :-/

Given the chart we've plotted recently (below), I wonder if we want to limit Hyrise to all physical cores and simply use this number for the client count as well. 
<img width="817" alt="image" src="https://user-images.githubusercontent.com/1745857/131676359-74f3cb5f-e560-43d4-83db-8ea0ed278621.png">

Reply:
I adjusted the file again. Is that what you meant?

Reply:
Alright, that properly works. On nemea we get `28` now. macOS is not out typical benchmarking OS but we could still simply divide the count by 2.

Reply:
/ 2 sounds good. Makes even sense for the ARM Macs with their energy efficient cores (at least for now).

Reply:
Exactly.